app/views/social_previews/comment.html.erb

```diff
@@ -66,84 +44,90 @@
     margin-left: 1vw;
     margin-right: 0.5vw;
   }
-
-  .preview-comment {
-    margin: 3vw auto;
+  .title-area {
+    position: absolute;
+    bottom: 12vw;
+    left: 0;
+    right: 0;
+    top: 0;
   }
-
-  .preview-comment h1 {
+  h2 {
     color: <%= dark_color %>;
-    width: 92%;
-    margin: 1vw auto;
-    font-family: "Helvetica Neue", Helvetica, Arial, "Lucida Grande", "Roboto", sans-serif;
+    width: 94%;
+    margin: 1.4vw auto;
+    font-size: 3.6vw;
   }
-
-  h2 {
+  h1 {
     color: <%= dark_color %>;
-    width: 92%;
+    width: 94%;
     margin: 1vw auto;
-    font-family: "Helvetica Neue", Helvetica, Arial, "Lucida Grande", "Roboto", sans-serif;
-    font-size: 3vw;
+    position: absolute;
+    padding-left: 2.5%;
+    top: 50%;
+    -ms-transform: translateY(-50%);
+    transform: translateY(-50%);
+    line-height: 1.05em;
   }
 
   .preview-user {
-    font-size: 3.5vw;
+    font-size: 3vw;
     position: absolute;
-    background: <%= accent_color %>;
+    color: <%= dark_color %>;
     left: 1.5vw;
-    bottom: 1.5vw;
-    padding: 1vw 3vw;
-    padding-left: 1.2vw;
+    bottom: 1.25vw;
+    padding: 0.8vw 2vw;
+    padding-left: 0.8vw;
     font-weight: 500;
-    border-radius: 100vw;
   }
 
   .preview-user img {
     border-radius: 100vw;
-    height: 5vw;
-    width: 5vw;
+    height: 4.5vw;
+    width: 4.5vw;
     vertical-align: -1.3vw;
     border: 0.25vw solid<%= dark_color %>;
   }
 
-  .preview-dev-logo {
+  .badge-images {
     position: absolute;
-    bottom: 2.1vw;
-    right: 2.1vw;
+    bottom: 1.4vw;
+    right: 1.5vw;
     text-align: right;
   }
 
-  .logo {
+  .badge-images img {
     height: 6vw;
+    right: 6vw;
     fill: <%= dark_color %>;
+    margin-left: 0.5vw;
+    transform: rotate(5deg);
   }
 </style>
+
 <div class="preview-div-wrapper">
-  <div class="circle circle1"></div>
-  <div class="circle circle2"></div>
-  <div class="circle circle3"></div>
   <div class="preview-div">
-    <% font_size = 12 - (((@comment.commentable.title.size**0.89) + 85) / 16) %>
-    <% if @comment.commentable.tag_list.include?("discuss") %>
-      <% font_size = 10.5 - (((@comment.commentable.title.size**0.89) + 85) / 16) %>
-      <div class="preview-info-header">
-        <% if @comment.commentable.comments_count > 0 %>
-          #discuss <img src="<%= asset_path "comments-bubble.png" %>" /><%= @comment.commentable.comments_count %>
-        <% else %>
-          New Discussion!
-        <% end %>
-      </div>
+    <% if @comment.title.size < 15 %>
+      <% font_size = 12 %>
+    <% elsif @comment.title.size < 30 %>
+      <% font_size = 9.2 %>
+    <% elsif @comment.title.size < 50 %>
+      <% font_size = 8.3 %>
+    <% else %>
+      <% font_size = 15 - ((((@comment.title).size**0.92) + 85) / 16) %>
```

Comment:
Same as in `_article.html.erb`, it's probably worth it to surface up to the controller